recipes/metilene/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: metilene
+  version: "0.2.6"
+
+about:
+  home: http://www.bioinf.uni-leipzig.de/Software/metilene/
+  license: GPLv2
+  license_file: LICENSE
+  summary: Fast and sensitive detection of differential DNA methylation
+
+build:
+  number: 0
+
+source:
+  fn: metilene_v02-6.tar.gz
+  url: "http://www.bioinf.uni-leipzig.de/Software/metilene/metilene_v02-6.tar.gz"
+  sha256: 832255cc58d6f3d4a1a157ca6a97186f0ff66a42480d3d0e45eceb4771f3978b
+  patches:
+    - patch.0
+
+requirements:
+  build:
+    - llvm # [osx]
+    - gcc # [linux]
+
+  run:
+    - perl-threaded
+    - bedtools >=2.24
+    - r-ggplot2 >=2.0.0
+    - libgcc # [linux]
+
+test:
+  commands:
+    - metilene --help 2>&1 | grep VERSION
+    - unset PERL5LIB
+    - unset PERL_LOCAL_LIB_ROOT
```

Comment:
No this should not be needed, and it does not guarantee that the tool is working after install, but only if one has set certain perl system variables to use local versions of modules.
These Variables like PERL5LIB, PERL_LOCAL_LIB_ROOT, PERL_CPANM_HOME are not handled correctly by conda, so even in an environment where I use conda-perl, the PERL5LIB still points to my local module dir, which results in a version clash. Bu unsetting these variables one can build and install and has to run the scripts with an additional -I $path_to_environment_perl_modules.
Not what I was expecting, but it seems a similar problem occurs to python users when setting their python module path to something not global, see [http://conda.pydata.org/docs/troubleshooting.html#conda-claims-installed](url).
The solution should not be to unset all these variables in my .bashrc, as this will render my setup completely useless and I still want to be able to use my system perl which is newer than the conda perl